core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/wordCloud/WordCloudOpFinalExec.java

```diff
@@ -20,14 +21,21 @@
  *
  */
 public class WordCloudOpFinalExec implements OperatorExecutor {
-    private final int MAX_FONT_SIZE = 200;
-    private final int MIN_FONT_SIZE = 50;
-    private HashMap<String, Integer> termFreqMap;
+    private static final int MAX_FONT_SIZE = 200;
+    private static final int MIN_FONT_SIZE = 50;
+
+    private List<Tuple> prevWordCloudTuples = new ArrayList<>();
+    private HashMap<String, Integer> termFreqMap = new HashMap<>();
+
     private static final Schema resultSchema = Schema.newBuilder().add(
             new Attribute("word", AttributeType.STRING),
             new Attribute("size", AttributeType.INTEGER)
     ).build();
 
+    public static final int UPDATE_INTERVAL_MS = 500;
```

Comment:
I see you defined `UPDATE_INTERVAL_MS` in `WordCloudOpPartialExec` (#958). Do we want to unify both configuration declarations into one?